### DISCUSSION TITLE
Burger Menu for mobile view?

### DISCUSSION THREAD

**Discussion by ghost** [linuxserver/Heimdall#716] [Ideas]
Hey, i got an iphone 8 (1334 × 750 Pixel, 326 ppi,) and i want to use heimdall as my startpage, but the menu on the right side is colliding with the tiles.
maybe that menu could be hidden and implementen as a burger menu on the upper right site of the page or something like that. would be great to use heimdall on smartphones without any disadvantage

**Comment by jin-ds** [comment]
Yes please!

**Comment by Rogue136198** [comment]
I would like to see an option to make the menu disappear altogether when an admin password is set. Make the main page completely clean and move configuration to a sub page such as /config end users (family) don't need anything shiny to distract them.

**Comment by NightyLive** [comment]
Dont want to be rude or anything like this but I'm quite suprised, this is like a basic design to have admin/editing functions on another layout than the main one that you want to keep clean as possible.
This tool is very usefull because I just want a lightwheight and basic welcome page where I can adds links. But now I'm considering migrating to something else since the users management is like "meh" and there is no way to have a clean page for users.
You can't even restrict the permission, the solution is to put a password and dont tell it to users but you can't using this solutions when using tags, you'll be disallowed to browse those tags.
Please let us just use a admin account managing the thing, and a basic public page with nothing else than the page itself.

If you want to remove the menus here is how to have a clean welcome page and a admin page, not the most advanced and secure thing, but it work :
PS: I'm not used to Laravel, I found this quick and dirty solution, they is certainly more efficients way to do it.

1. At end of "/Heimdall/routes/web.php" add those lines :
```
Route::get ('/admin', function () {
    return view('admin');
});
```
2. Copy "/Heimdall/resources/views/layouts/app.blade.php" to "/Heimdall/resources/views/admin.blade.php"

3. Edit and remove those two blocs from "/Heimdall/resources/views/layouts/app.blade.php" :

```
                    @if($allusers->count() > 1)
                    <div id="switchuser">
                        @if($current_user->avatar)
                        <img class="user-img" src="{{ asset('/storage/'.$current_user->avatar) }}" />
                        @else
                        <img class="user-img" src="{{ asset('/img/heimdall-icon-small.png') }}" />
                        @endif
                        {{ $current_user->username }}
                        <a class="btn" href="{{ route('user.select') }}">Switch User</a>
                    </div>
                    @endif
```
```
                    <div id="config-buttons">


                        @if(Route::is('dash') || Route::is('tags.show'))
                        <a id="config-button" class="config" href=""><i class="fas fa-exchange"></i></a>
                        @endif

                        <a id="dash" class="config" href="{{ route('dash', []) }}"><i class="fas fa-th"></i></a>
                        @if($current_user->id === 1)
                        <a id="users" class="config" href="{{ route('users.index', []) }}"><i class="fas fa-user"></i></a>
                        @endif
                        <a id="items" class="config" href="{{ route('items.index', []) }}"><i class="fas fa-list"></i></a>
                        <a id="folder" class="config" href="{{ route('tags.index', []) }}"><i class="fas fa-tag"></i></a>
                        <a id="settings" class="config" href="{{ route('settings.index', []) }}"><i class="fas fa-cogs"></i></a>
                    </div>
```

The main page is now clean, if you want to administer Heimdall you might can do it by using this url : yourheimdall.site/admin

Cons :
- Heimdall will redirect you to the homepage and you need to return manualy to the admin page after each changes.
- This is NOT secured, I use an account with no password to let homepage and tags available to anyone, but if users know the admin url they can change things.
- The admin page doesnt show the apps

**Comment by NightyLive** [comment]
Did some updates to my last comment with a proper solution.

**Comment by k-matti** [comment]
Is menu panel still undisclosed?

**Comment by tom666-debug** [comment]
@NightyLive 
thanks for your "quick and dirty solution" above, this was very helpful for me
